blurb/blurb.py

```diff
@@ -766,9 +778,23 @@ def find_editor():
             found_path = shutil.which(fallback)
         if found_path and os.path.exists(found_path):
             return found_path
+
+    # For simplicity, recommend the old EDITOR environment varibale. Advanced
+    # users should consult the documention if they want to use more specific
+    # confguration.
     error('Could not find an editor! Set the EDITOR environment variable.')
 
 
+def git_core_editor():
```

Comment:
I prefer the opposite, putting helpers at the bottom, but I can move the function before find_editor if you like.

I'm pretty sure about the keeping the logic in a function, the way to get git configuration is not interesting to the editor lookup code.